nilearn/datasets/func.py

```diff
@@ -98,6 +98,14 @@ def fetch_haxby(data_dir=None, n_subjects=1, fetch_stimuli=False,
     n_subjects: int, optional
         Number of subjects, from 1 to 6.
 
+        NOTE: n_subjects is deprecated from 0.2.5.1 and will be removed in 0.3
+        Use `subjects` instead.
+
+    subjects : int or list, optional
```

Comment:
To clarify myself, only subjects should there ? and it should accept only list otherwise raise Error or warning and then convert to list ? ping @AlexandreAbraham @bthirion